### DISCUSSION TITLE
Automatic switching to device screen after sending print to printer v1.5.0

### DISCUSSION THREAD

**Discussion by bluecobras09** [OrcaSlicer/OrcaSlicer#548] [Q&A]
Since moving to OrcaSlicer v1.5.0, when I send a plate to the printer, OrcaSlicer does not automatically go to the Device page.  Was this a change?  I checked the release notes and didn't find anything.  Or is something wrong with my install?

Thanks!

**Comment by kylek29** [comment]
I have the same issue in 1.5 as well.
